_posts/2016-08-28-katello-puppet4-integration.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: Katello Puppet4 integration
+date: 2016-08-28
+author: Zach Huntington-Meath
+tags:
+- katello
+- puppet 4
+excerpt: |
+    Puppet 4 integration is coming to Katello and we wanted to share with you
+    how it works!
+---
+---
+layout: post
+title: Katello Puppet4 integration
+date: 2016-09-19
+author: Zach Huntington-Meath
+tags:
+- katello
+- puppet 4
+excerpt: |
+    Puppet 4 integration is coming to Katello and we wanted to share with you
+    how it works!
+---
+
+### Puppet 4 Integration with Katello
+
+Recently Foreman has grown to work with Puppet 4. Katello is following suite and will soon be ready to use Puppet 4. In this post we will explain how to create new Katello environments Puppet 4 ready, and upgrade exisiting systems.
```

Comment:
We are ready for Puppet 4! Maybe indicate that by changing the second sentence to note that it is in nightly and will be included in 3.2 ?